src/test/java/org/testeditor/fixture/web/HelperTool.java

```diff
@@ -0,0 +1,228 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.web;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelperTool {
+
+	private static final Logger logger = LoggerFactory.getLogger(HelperTool.class);
+	private static final String PROPERTIES_FILE = "test.properties";
+	private String proxyValue = null;
+	private String proxyPort = null;
+	private String proxyUserValue = null;
+	private String proxyPasswordValue = null;
+	private String proxyNonProxyValue = null;
+	private String userName = null;
+	private String passwd = null;
+	private String url = null;
+
+	public void initializeProperties() {
+		this.loadProperties();
+		this.setProxyValuesForFirefox();
+	}
+
+	private void setProxyValuesForFirefox() {
```

Comment:
Name is misleading, how about exportToSystemProperties. If more than the firefox will be adressed, additional parameters can be passed (like BrowserProperties.FirefoxPropKeys or something like that)